pkg/jwt/parse.go

```diff
@@ -0,0 +1,44 @@
+package jwt
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/lestrrat-go/jwx/jwk"
+	"github.com/lestrrat-go/jwx/jwt"
+	"github.com/pkg/errors"
+)
+
+// rawJWTFromAuthHeader returns a raw jwt token from the auth header
+func rawJWTFromAuthHeader(r *http.Request) string {
```

Comment:
If this function returned a bool and a string, you could more easily check that the returned string isn't just empty, if that's something that's important here.

Reply:
It will end up failing with an error here if its empty https://github.com/Chia-Network/go-modules/pull/1/files#diff-1951904f8106e57a95a3ce0bcd8de0d2cbd07a84d765eed182c6b1d03f11cbb9R32

I could return a bool or something, but then its more code everywhere downstream we want to use it, to check bool, then use.. I'm kind of inclined to just let the actual jwt parsing fail on the empty string.

Reply:
```ParseTokenFromHeaderWithJWKS():jwt.ParseString(): failed to parse token data as JWS message: invalid compact serialization format: invalid number of segments```